rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/RosettaExpressionFormatter.xtend

```diff
@@ -175,15 +255,23 @@ class RosettaExpressionFormatter extends AbstractRosettaFormatter2 {
 	private def dispatch void unsafeFormatExpression(RosettaConditionalExpression expr, extension IFormattableDocument document, FormattingMode mode) {
 		val extension conditionalGrammarAccess = rosettaCalcConditionalExpressionAccess
 		
+		// fix edge case where 'then' inside constructor value is not indented correctly
+		if (expr.eContainer instanceof ConstructorKeyValuePair) {
```

Comment:
I guess this "check" is not necessary anymore, and you should just call `surroundIndent...`?